content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -39,7 +39,7 @@ namespace.
 
 DNS queries may be expanded using the pod's `/etc/resolv.conf`. Kubelet 
 sets this file for each pod. For example, a query for just `data` may be 
-expanded to `data.test.cluster.local`. The values of the `search` option 
+expanded to `data.test.svc.cluster.local`. The values of the `search` option 
```

Comment:
the resolv.conf, as you can see it some lines below, have
```
search <namespace>.svc.cluster.local svc.cluster.local cluster.local
```

that means that it will append all those domains to the query, so `data.test.cluster.local` is a possibility

However, the records for a Service are defined in the DNS specification

I think that this change is correct, but I prefer @chrisohaver to chime in

https://github.com/kubernetes/dns/blob/master/docs/specification.md#23---records-for-a-service-with-clusterip
```
<service>.<ns>.svc.<zone>. <ttl> IN AAAA <cluster-ip>
```

Reply:
> so `data.test.cluster.local` is a possibility

Not if the searched host was `data`, as is in this example.  That last search domain would just produce `data.cluster.local`.

The PR is correct.

Reply:
ups, right, I was thinking in `data.test` :upside_down_face: